assets/models/system/mmd-3x-rtmdet-ins_x_8xb16-300e_coco/spec.yaml

```diff
@@ -0,0 +1,123 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: mmd-3x-rtmdet-ins_x_8xb16-300e_coco
+path: ./
+properties:
+  SharedComputeCapacityEnabled: true
+  SHA: 4d77feb7a8e5967686d9bf922d2bb61162de57d2
```

Comment:
I thought we are not keeping SHA anymore, are we keeping it?

Reply:
Just discussed it with @vadthyavath. We are keeping it as of now as per the design doc. 
https://microsoftapc-my.sharepoint.com/:w:/r/personal/ayushmishra_microsoft_com/Documents/System%20Registry%20model%20card%20alignment.docx?d=w2e13db38a1a94148b1dcac14a76e1f72&csf=1&web=1&e=4kS3VU